### ISSUE TITLE
Login error 405 (related to #105)

### ISSUE THREAD

**Issue by isene** [bitlbee/bitlbee-facebook#195]
Digital Ocean had an emergency reboot of my droplet server, so I had to reconstruct my weechat setup (with bitlbee). I got my IRC channels up and Slack, Twitter and Discord via Bitlbee works just fine. But not Facebook.

I run Ubuntu 16.04, Weechat 2.8, BitlBee 3.6-1+20200507+master+26-gb8b931da-git , bitlbee-facebook 20190203~c76b36b~331.

I have done:
```
account facebook del
account add facebook <user> <password>
account facebook on

```

```
@root │ facebook - Logging in: Authenticating
  `-> │ facebook - Login error: User must verify their account on www.facebook.com (405)
  `-> │ facebook - Logging in: Signing off..
  `-> │ facebook - Logging in: Reconnecting in 405 seconds..

```
I then go to facebook and click YES to verify that it was me logging in from an unusual location. But the same thing happens when it tries to log in again.

I have also tried a facebook app password, but the same thing happens. I have also tried to log in to facebook directly from the server with w3m and verified that the login is indeed me (going through sending of a 6 digit code). But that didn't help.

I am stuck.

**Comment by dequis** [comment]
Yeah digital ocean is probably banned. You'll have to find another IP range.

**Comment by isene** [comment]
But it surely wasn't banned overnight just as my server went down for a few hours, or?

**Comment by Linkandzelda** [comment]
Just adding that I've tried 3 different IPs, even the one which is the same as my VPN that I used on my machines to login to facebook, and this issue still happens. Been having the issue since May.

**Comment by isene** [comment]
Since I opened this issue, I have not been able to log in once. I have given up on using bitlbee-facebook - unless someone reports revolutionary news here..

**Comment by rathann** [comment]
This is happening for me even if I'm running bitlbee on the same machine as my browser. Looks like Facebook banned all third-party apps even when using a generated application password for Bitlbee.

**Comment by TRPB** [comment]
FYI, libpurple-facebook still works in pidgin so it's not all third party apps that are blocked.

**Comment by gmartsenkov** [comment]
@TRPB it does indeed, i decided to try and use FB_ORCA_AGENT from the pidgin library and voila, it fixed it; I have a PR open https://github.com/bitlbee/bitlbee-facebook/pull/213. You can compile the branch from my fork and test it out

**Comment by blacklight** [comment]
@gmartsenkov your user agent trick fixed the issue for me - thanks!
